docs/serverless/08-02-create-git-function.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Create a Function from Git repository sources
+type: Tutorials
+---
+
+Create a sample Function from code and dependencies stored in a Git repository. The tutorial is based on the Function from the [`orders service` example](https://github.com/kyma-project/examples/tree/master/orders-service). It shows how you can fetch Function's source code and dependencies from a public Git repository that does not require any authentication method.
+
+> **NOTE:** Read more about [Git source type](#details-git-source-type) and different ways of securing your repository.
+
+## Steps
+
+Follows these steps:
+
+<div tabs name="steps" group="create-function">
+  <details>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+1. Export these variables:
```

Comment:
Sth is wrong with formatting: 
![image](https://user-images.githubusercontent.com/40655785/92724631-41403a80-f36b-11ea-878e-91bdef67fe2e.png)